__DEFINES/setup.dm

```diff
@@ -962,7 +963,7 @@ var/list/RESTRICTED_CAMERA_NETWORKS = list( //Those networks can only be accesse
 
 #define IS_SLOW 256
 #define IS_PLANT 512
-#define IS_WHITELISTED 1024
```

Comment:
"This PR also removes the IS_WHITELISTED species flag, because it had no meaningful use anyway and room for the new species flag was needed."